syntax = "proto3";

service Showtime {
    rpc GetShowTimes (Empty) returns (stream ScheduleData) {};
    rpc GetMoviesByDate (Date) returns (ScheduleData) {};
}

message ScheduleData {
    string date = 1;
    repeated string MovieDatas = 2;
}

message Date {
    string date = 1;
}

message Empty {
}